dev-requirements/dev.in

```diff
@@ -0,0 +1,5 @@
+-r base.in
+-r test.in
+
+isort
```

Comment:
This should probably go in test.in and be checked using Travis, eventually.

Run with e.g. `isort --verbose --check-only --diff --recursive src tests setup.py`

Configure in `setup.cfg` e.g.
```
[isort]
force_single_line = True
line_length = 120
known_first_party = fccodegen
default_section = THIRDPARTY
not_skip = __init__.py
```